csvcubed/tests/behaviour/inspectcli.feature

```diff
@@ -0,0 +1,122 @@
+            http://purl.org/linked-data/sdmx/2009/dimension#refPeriod                                Dimension           Period     true
+            http://gss-data.org.uk/def/trade/property/dimension/alcohol-type                         Dimension     Alcohol Type     true
+            ./alcohol-bulletin.csv#dimension/alcohol-sub-type               Alcohol Sub Type         Dimension Alcohol Sub Type     true
+            ./alcohol-bulletin.csv#dimension/alcohol-content                 Alcohol Content         Dimension  Alcohol Content     true
+            ./alcohol-bulletin.csv#dimension/clearance-origin               Clearance Origin         Dimension Clearance Origin     true
+            http://purl.org/linked-data/cube#measureType                                             Dimension     Measure Type     true
+            http://gss-data.org.uk/def/measure/alcohol-duty-receipts                                   Measure                      true
+            http://gss-data.org.uk/def/measure/beer-duty-receipts                                      Measure                      true
+            http://gss-data.org.uk/def/measure/cider-duty-receipts                                     Measure                      true
+            http://gss-data.org.uk/def/measure/clearances                                              Measure                      true
+            http://gss-data.org.uk/def/measure/clearances-of-alcohol                                   Measure                      true
+            http://gss-data.org.uk/def/measure/production-volume                                       Measure                      true
+            http://gss-data.org.uk/def/measure/production-volume-alcohol                               Measure                      true
+            http://gss-data.org.uk/def/measure/spirits-duty-receipts                                   Measure                      true
+            http://gss-data.org.uk/def/measure/wine-duty-receipts                                      Measure                      true
+            http://purl.org/linked-data/sdmx/2009/attribute#unitMeasure                              Attribute             Unit     true
+            http://purl.org/linked-data/sdmx/2009/attribute#obsStatus                                Attribute           Marker    false
+        """
+        And the Code List Printable should be
+        """
+        - The data cube has the following code lists:
+                                             Code List   Code List Label  Columns Used In
+        ./alcohol-sub-type.csv#scheme/alcohol-sub-type                   Alcohol Sub Type
+          ./alcohol-content.csv#scheme/alcohol-content                    Alcohol Content
+        ./clearance-origin.csv#scheme/clearance-origin                   Clearance Origin
+        """
+
+    Scenario: inspect cli should produce printable for code list metadata json-ld input
```

Comment:
I'd avoid using `cli` in the scenario descriptions since you're not actually calling the CLI here but the functions under the hood.

Reply:
This is now updated.